package.json

```diff
@@ -14,7 +14,6 @@
     "clean-all": "yarn clean && rimraf output",
     "load": "sourcecred load",
     "graph": "sourcecred graph",
-    "score": "sourcecred score",
     "site": "sourcecred site",
     "serve": "sourcecred serve",
     "grain": "sourcecred grain"
```

Comment:
We could add a npm script called `go` that runs `sourcecred go`, instead of running it directly in the action.

Reply:
Done 🙂